### DISCUSSION TITLE
Update time

### DISCUSSION THREAD

**Discussion by KA19XFinntage** [fsaris/home-assistant-zonneplan-one#138] [Q&A]
Is there a way how i can make it update faster instead of every hour

**Comment by fsaris** [comment]
Most values are updated every 5min. 
Gas related values depending on your own meter only once a hour.

What sensors are you looking at?

**Reply by AMensink** [reply]
I am looking for sensor.zonneplan_current_usage to update faster than 5 minutes.
But it may well be that Zonneplan does not update more often?
My P1 update every 10 seconds.

**Reply by fsaris** [reply]
The zonneplan api will not update the values more often. To get more values you will need to read the P1 using some other connector/tool
